ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact/cucushift-installer-rehearse-aws-ipi-compact-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-compact
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-compact-provision
```

Comment:
Use existing `cucushift-installer-rehearse-aws-ipi-[de]provision` instead of introducing a new chain.

Reply:
OK, I will use the existing chain.Thank you.